scripts/image/test-all-outbound-connectivity-scenarios.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+mkdir -p python-output
+mkdir -p pcaps
+
+start_timestamp_seconds=$(date +%s)
+
+./test-outbound-connectivity.sh --cleanup baseline
+./test-outbound-connectivity.sh --cleanup missing-models
+./test-outbound-connectivity.sh --cleanup offline
+./test-outbound-connectivity.sh --cleanup offline-and-missing-models
+
+set +e
+found_pcap_files=$(find "pcaps" -maxdepth 1 -name "*.pcap" -type f -newermt "@$start_timestamp_seconds" 2>/dev/null | wc -l | tr -d ' ')
```

Comment:
This might be a bash version issue, but I get an error running `find "pcaps" -maxdepth 1 -name "*.pcap" -type f -newermt "@$start_timestamp_seconds"` that it can't parse the date. I was able to get it to work by changing line 8 to `start_timestamp_seconds=$(date)` and removing the `@` from the `find` commands.